cucumber/api/features/support/rest_helpers.rb

```diff
@@ -144,6 +145,13 @@ def authn_local_request command
     end
   end
 
+  def authn_request(url:, api_key:, encoding:, can:)
```

Comment:
looks like `encoding` is actually a boolean and not an encoding value. i think that `accept_base64_encoding` is better.

Reply:
Not a boolean anymore